include/RAJA/exec-openmp/scan_openmp.hxx

```diff
@@ -82,6 +82,7 @@ void inclusive_scan_inplace(const omp_parallel_for_exec&,
                             ValueT v)
 {
   using Value = typename std::decay<decltype(*std::declval<Iter&>())>::type;
+  // using Value = typename std::iterator_traits <Iter>::value_type;
```

Comment:
No idea on this, I'll wait on @trws for it. We're also bringing in Nvidia folks for the RAJA meeting, we might be able to ask them about it